event_product/README.rst

```diff
@@ -0,0 +1,119 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+==============
+Event Products
+==============
+
+This module extends the functionality of events and products to support
+combining them and allow you to have a *catalog* of events that you organize.
```

Comment:
Current sentence is weird and not so clear at first. What about:
This module extends the functionality of events and products allowing their combination and the organization of a *catalog* of events.